src/services/helpers.ts

```diff
@@ -1,5 +1,4 @@
 import { AxiosResponse } from 'axios'
-import normalizeStrings from 'normalize-strings'
```

Comment:
It looks to me like we can then remove the package 'normalize-strings' from our code. Can you check if that is indeed the case?

Reply:
yes, we don't use it anymore. I will remove it.

Reply:
seems like the changes related to  package removal disappeared after committing it. There is no package on my local repository